qiskit_ibm_runtime/options/resilience_options.py

```diff
@@ -34,115 +34,63 @@
     "QuarticExtrapolator",
 ]
 
-ZneExtrapolatorType = Literal[
-    "linear",
-    "exponential",
-    "double_exponential",
-]
-
 
 @primitive_dataclass
 class ResilienceOptionsV2:
-    """Resilience options.
+    """Resilience options for V2 Estimator.
 
     Args:
-        measure_noise_mitigation: Whether to enable measurement error mitigation method.
-            By default, this is enabled for resilience level 1, 2, and 3 (when applicable).
+        measure_mitigation: Whether to enable measurement error mitigation method.
+            Further suboptions are available in :attr:`~measure_noise_learning`.
+            Default: True.
+
+        measure_noise_learning: Additional measurement noise learning options.
+            See :class:`MeasureNoiseLearningOptions` for all options.
 
         zne_mitigation: Whether to turn on Zero Noise Extrapolation error mitigation method.
-            By default, ZNE is enabled for resilience level 2.
+            Further suboptions are available in :attr:`~zne`.
+            Default: False.
 
-        zne_noise_factors: An list of real valued noise factors that determine by what amount the
-            circuits' noise is amplified.
-            Only applicable if ZNE is enabled.
+        zne: Additional zero noise extrapolation mitigation options.
+            See :class:`ZneOptions` for all options.
 
-        zne_extrapolator: An extrapolation strategy. One or more of ``"exponential"``,
-            ``"double_exponential"``, and ``"linear"``.
-            Only applicable if ZNE is enabled.
+        pec_mitigation: Whether to turn on Probabilistic Error Cancellation error mitigation method.
```

Comment:
Since pec and zne mitigation are mutually exclusive, why not have a single option like on the server side that has to be set to PEC or ZNE?

Reply:
Two reason: (1) it's easier to code, like you don't need to know which methods are available, and (2) it matches the other switches, like `measure_mitigation` and `dynamical_decouplig.enable`. 

There is a check to verify they're not both specified.